connectors/sources/mssql.py

```diff
@@ -143,9 +142,28 @@ async def close(self):
             return
         self.connection.close()
 
+    def get_pem_format(self, keys, max_split=-1):
```

Comment:
Why create a new function instead of updating the old one?

Reply:
As customers are waiting for MS-SQL, we have updated it at the connector level so other connectors do not get affected. Also, we are working on it. Once testing is done for all the connectors, we will raise a follow-up PR in which we move these changes to utils and remove them from MS-SQL.